File: examples/geocoding/reverse/node/reverseGeocodeLocation.js
const { GeocodingService } = require("m3o/geocoding");

// Reverse lookup an address from gps coordinates
async function reverseGeocodeLocation() {
  let geocodingService = new GeocodingService(process.env.MICRO_API_TOKEN);
  let rsp = await geocodingService.reverse({
    latitude: 51.5123064,
    longitude: -0.1216235,
  });
  console.log(rsp);
}

reverseGeocodeLocation();

File: examples/geocoding/reverse/go/reverseGeocodeLocation.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/geocoding"
)

// Reverse lookup an address from gps coordinates
func ReverseGeocodeLocation() {
	geocodingService := geocoding.NewGeocodingService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := geocodingService.Reverse(&geocoding.ReverseRequest{
		Latitude:  51.5123064,
		Longitude: -0.1216235,
	})
	fmt.Println(rsp, err)
}
